src/libs/SearchUtils.ts

```diff
@@ -138,6 +141,7 @@ function getTransactionsSections(data: OnyxTypes.SearchResults['data'], metadata
                 formattedMerchant,
                 date,
                 shouldShowMerchant,
+                searchHash: context.searchHash,
```

Comment:
Hmm I'm not a fan of passing this data to every single item. Could we retrieve it globally somehow?

Reply:
Yes I know. I guess react context is the only other way that comes to my mind. We don't have like "global state" in the app other than onyx, and Im not sure I want to set inside Onyx - feels like onyx is more for backend data.
I will experiment with context and see how this looks

Reply:
I left a comment above about removing the hash from the URL and just computing it from the query whenever we need it. I don't know if that'd remove the need to pass the hash around though